### DISCUSSION TITLE
Password / Token in URL for instant access without manual input

### DISCUSSION THREAD

**Discussion by sevetseh28** [coder/code-server#6862] [Q&A]
Hi, I was wondering if there is any way to embed the Password / Token into the URL so that there is no login page / manual user intervention to get access to the IDE, but keep it secured under a password. OpenVSCode Server does this by embedding `tkn` in the URL and then the server will set it in the Cookie. 

Thank you!

**Comment by code-asher** [comment]
Not at the moment but there are some open issues about it:
https://github.com/coder/code-server/issues/6733
https://github.com/coder/code-server/issues/5819
